.github/workflows/test.yml

```diff
@@ -7,50 +7,213 @@ on:
       - master
       - develop
 
+concurrency:
+  group: axe-core Tests ${{ github.ref_name }}
+  cancel-in-progress: ${{ github.event_name != 'push' }}
+
+permissions: {}
```

Comment:
Nifty trick for disabling all permissions in one sweep. So we refine permissions needed (if any) at the job level.

Ref: https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#defining-access-for-the-github_token-scopes